src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getDefaultFigure() {
+        Circle circle = new Circle();
+        circle.setColor(Figure.defaultColor);
+        circle.setRadius(10);
+        return circle;
+    }
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int indexOfFigure = random.nextInt(5);
```

Comment:
Consider defining a constant for the number of figure types instead of using the magic number `5`. This will improve code readability and maintainability .